pyvista/plotting/plotting.py

```diff
@@ -3644,6 +3648,10 @@ def add_legend(self, labels=None, bcolor=(0.5, 0.5, 0.5), border=False,
             for i, (text, color) in enumerate(labels):
                 self.legend.SetEntry(i, legendface, text, parse_color(color))
 
+        if origin:
+            assert len(origin) == 2, 'origin must be of length 2'
```

Comment:
We try to avoid `assert` statements within the library
```suggestion
            if len(origin) != 2:
                raise ValueError('`origin` must be of length 2. Given value is of length {}'.format(len(origin)))
```

Reply:
> We try to avoid assert statements within the library

Sure, I was just being lazy :wink: